resources/lang/en/detail.php

```diff
@@ -0,0 +1,5 @@
+<?php
+return [
+    'author'=>"written by: ",
```

Comment:
```suggestion
    'author' => "written by: ",
```
fix tương tự các chỗ khác nha